src/blocks/scratch3_looks.js

```diff
@@ -244,6 +244,7 @@ class Scratch3LooksBlocks {
             looks_thinkforsecs: this.thinkforsecs,
             looks_show: this.show,
             looks_hide: this.hide,
+            looks_hideallsprites: this.doNothing,
```

Comment:
Instead of defining a `doNothing` member in different categories, I would be okay with just replacing that with `() => {} // legacy no-op block`

Reply:
Cool, will update this and the scroll-blocks PR.